DomainContent/AvatarStore/CheckoutZone/checkoutScanner.js

```diff
@@ -0,0 +1,77 @@
+//
+//  checkoutScanner.js
+//
+//  Created by Rebecca Stankus on 10/10/17.
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This script acts on the scanner zone to pull up an item's marketplace page to enable purchasing.
+(function() {
+    
+    var TABLET = Tablet.getTablet("com.highfidelity.interface.tablet.system");
+    var MARKET_PLACE_ITEM_URL_PREFIX = 'https://metaverse.highfidelity.com/marketplace';
+    var MARKETPLACES_INJECT_SCRIPT_URL = ScriptDiscoveryService.defaultScriptsPath + "/system/html/js/marketplacesInject.js";
+    var SCAN_RADIUS = 0.15; // meters
+    var OVERLAY_PREFIX = 'MP';
+    
+    var interval;
+    var properties;
+    var checkoutZoneID;
+
+    var scannedMPOverlays = {};
+
+    var onEntityAdded = function(entityID) {
```

Comment:
I would suggest we remove the print statements or wrap them in a debug flag

Reply:
removed unnecessary print statements